### DISCUSSION TITLE
Suggestion: move the "Security Bug Bounty Hunter" highlight to Achievements

### DISCUSSION THREAD

**Discussion by xdumaine** [community/community#28383] [Other Feature Feedback, Questions, & Ideas]
The new Achievements section seems like the perfect place to highlight the Security Bug Bounty Hunter achievement.

![image](https://user-images.githubusercontent.com/833911/182930854-704051bd-094f-4e9b-bb49-56b6853d1ab0.png)

**Comment by radar-blip** [comment]
You mean the Security Bug Bounty Hunter Highlight 😄
